### PR TITLE
Frontend turns

### PR DESCRIPTION
**Pull request by xKoem**
# PR for self review

Frontend functionality for turns
Game is now started automatically

### CODE REVIEW

File: REST/src/main/java/pl/jakubowskiprzemyslaw/REST/services/FrontendMessageSenderService.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+@Component
+public class FrontendMessageSenderService {
+
+    private final SimpMessagingTemplate template;
+
+    public FrontendMessageSenderService(SimpMessagingTemplate template) {
+        this.template = template;
+    }
+
+    public void sendMessage(String playerName, String message) {
```
Comment (JaroslawSlaby) [inline REST/src/main/java/pl/jakubowskiprzemyslaw/REST/services/FrontendMessageSenderService.java]: sendTurn

File: REST/src/main/resources/templates/playing.html [hunk 2]
```diff
@@ -25,6 +23,6 @@ <h1>Best shooting frontend ever!</h1>
 <script src="/webjars/stomp-websocket/stomp.min.js"></script>
 <script type="text/javascript" src="/js/board.js"></script>
 
-<script>setInterval(refresh, 500)</script>
+<!--<script>setInterval(refresh, 500)</script>-->
 </body>
 </html>
```
Comment (JaroslawSlaby) [inline REST/src/main/resources/templates/playing.html]: new line

File: REST/src/main/java/pl/jakubowskiprzemyslaw/REST/controllers/PlayerConfigController.java [hunk 3]
```diff
@@ -42,7 +40,7 @@ public String sendGameConfig(@ModelAttribute("player") Player player, HttpServle
         sessionService.addObjectToSessionRequest(request, player);
 
         PlayerConfiguration playerConfiguration = new PlayerConfiguration(player);
-        queueService.sendObjectToQueue(playerConfigQueueName, playerConfiguration);
+        queueService.sendObjectToQueue(Queues._1PlayerRegistrationQueue, playerConfiguration);
```
Comment (JaroslawSlaby) [inline REST/src/main/java/pl/jakubowskiprzemyslaw/REST/controllers/PlayerConfigController.java]: As I remember, I've made this mistake, sorry

File: REST/src/main/resources/templates/playing.html (hunk 2 above)
Comment (JaroslawSlaby) [reply REST/src/main/resources/templates/playing.html]: and don't forget to cut all `console.log()` usages

File: REST/src/main/resources/templates/playing.html [hunk 4]
```diff
@@ -25,6 +23,6 @@ <h1>Best shooting frontend ever!</h1>
 <script src="/webjars/stomp-websocket/stomp.min.js"></script>
 <script type="text/javascript" src="/js/board.js"></script>
 
-<script>setInterval(refresh, 500)</script>
+<!--<script>setInterval(refresh, 500)</script>-->
```
Comment (JaroslawSlaby) [inline REST/src/main/resources/templates/playing.html]: We don't need this refreshing anymore (I hope so)

File: REST/src/main/resources/static/js/board.js [hunk 5]
```diff
@@ -55,98 +54,60 @@ function setCellAttribute(cell, attribute, myBoard) {
...
-        //disable clicking
-        startTimer(120, timerDiv);
     }
 }
 
-var userName;
-
-function connect() {
+function createQueuesConnection() {
     var socket = new SockJS('/timer-websocket');
```
Comment (JaroslawSlaby) [inline REST/src/main/resources/static/js/board.js]: We have to change SockJS parameter from `/timer-websocket` to something different, more descriptive and matching

File: REST/src/main/resources/static/js/board.js [hunk 6]
```diff
@@ -55,98 +54,60 @@ function setCellAttribute(cell, attribute, myBoard) {
     }
 }
 
-var timerDiv = document.getElementById("timer");
+function playerOnClick() {
```
Comment (JaroslawSlaby) [inline REST/src/main/resources/static/js/board.js]: @xKoem you've been right, clickMe() was a terrible name for this function :) 👍

File: REST/src/main/resources/templates/playing.html (hunk 4 above)
Comment (xKoem) [reply REST/src/main/resources/templates/playing.html]: yup, but I commented it and forgot to remove it before commiting 🗡

Comment (JaroslawSlaby) [reply REST/src/main/resources/templates/playing.html]: don't worry, tommorow we're going to kill this line with fire ;D

File: REST/src/main/java/pl/jakubowskiprzemyslaw/REST/services/FrontendMessageSenderService.java (hunk 1 above)
Comment (xKoem) [reply REST/src/main/java/pl/jakubowskiprzemyslaw/REST/services/FrontendMessageSenderService.java]: sendPlayingPlayer

File: REST/src/main/java/pl/jakubowskiprzemyslaw/REST/controllers/BoardsHandlerController.java [hunk 7]
```diff
@@ -1,68 +0,0 @@
...
-
-        boolean isPlayerPresent = currentPlayer.isPresent();
-        if (isPlayerPresent && currentPlayer.get().equals(sessionPlayer)) {
-            node = jsonService.createJSONNodeForPlayerShots(optionalBoardsView.get());
-            boardsViewHandler.clear();
-        }
-
-        return node;
-    }
-}
```
[the remark this answers is not in the document]
Comment (xKoem) [reply REST/src/main/java/pl/jakubowskiprzemyslaw/REST/controllers/BoardsHandlerController.java]: (y)

File: REST/src/main/resources/static/js/board.js (hunk 5 above)
Comment (xKoem) [reply REST/src/main/resources/static/js/board.js]: sure

**Pull request merged**